.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-.DS_Store
\ No newline at end of file
+.DS_Store
+coverage
```

Comment:
What is `coverage` coming from?

Reply:
Coverage is just the vitest code coverage that I added while part of these changes. I wanted to make sure that I covered all branches while writing my tests.